src/MSBuild.Sdk.SqlProj/Sdk/Sdk.targets

```diff
@@ -112,7 +112,17 @@
     <ItemGroup> 
       <!-- Resolve all package references to their physical location first -->
       <_ResolvedPackageReference Include="%(PackageReference.Identity)">
-        <DacpacFile>$([System.String]::new('$(NuGetPackageRoot)%(PackageReference.Identity)/%(PackageReference.Version)').ToLower())/tools/%(Identity).dacpac</DacpacFile>
+        <!-- Determine technical name of package (ie. DeVries.WMS.Database -> DeVries_WMS_Database) -->
```

Comment:
I'm guessing you got this comment from my original commit since this is referring to an internal package here. We should probably replace `DeVries.WMS.Database` here with something like `Foo.Bar.Database` or something.

Reply:
Okay, sure. I wasn't sure if it was an example or a real-world use case, although I suppose it is quite specific 😄